CDH-Studio/UpSkill
# IE limitation workaround

## What does this PR do
Internet explorer does not like writing to the value document.body.style. To fix this we leave the background as a plain white instead of the gradient pattern.

## Links to Jira
https://jira.ised-isde.canada.ca/browse/STUDIOUP-3

## Steps to Test
  - Run in IE and ensure the website works but with a plain white background
  - Run in another browser and ensure the website works with the gradient pattern background
